dotnet/Directory.Packages.props

```diff
@@ -5,6 +5,8 @@
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
   </PropertyGroup>
   <ItemGroup>
+    <PackageVersion Include="AWSSDK.BedrockRuntime" Version="3.7.306.1" />
```

Comment:
Can this be bumped to the latest version? Version 3.7.400 had some rework for observability which would be good to add in here.